website/pages/changelog.tsx

```diff
@@ -0,0 +1,25 @@
+import { parseMarkdownFile } from "@docusaurus/utils"
+import path from "path"
+import React from "react"
+import { MDXLayoutProvider } from "../layouts/mdx"
+import ChangelogContent from "../../CHANGELOG.md"
+
+// DOM code
+function Changelog({ content }) {
```

Comment:
Also `content` is not being used

Reply:
Thanks Lazar. Removing the unused imports.